pgtrigger/core.py

```diff
@@ -31,6 +34,31 @@
 _ignore = threading.local()
 
 
+def _get_database(model):
+    """
+    Obtains the database used for a trigger / model pair. The database
+    for the connection is selected based on the write DB in the database
+    router config.
+    """
+    return router.db_for_write(model) or 'default'
```

Comment:
Should use `from django.db import DEFAULT_DB_ALIAS`

Reply:
Good call! Will fix